projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-checkout.ts

```diff
@@ -495,7 +495,7 @@ export function reviewB2bOrderConfirmation(
     });
 
     if (!replenishment) {
-      cy.get('.cx-summary-card:nth-child(2) .cx-card').within(() => {
+      cy.get('.cx-summary-card:nth-child(2)').within(() => {
```

Comment:
yes, was failing due to several cx-card elements inside 'cx-summary-card:nth-child(2)'.
It is a new restriction for 'within' on Cy12 https://docs.cypress.io/guides/references/migration-guide#within

because 'cx-summary-card:nth-child(2)' is ensure to be a signle element, it fixes the prob, the DOM area is now slighly larger but does not cause any side effect with the inner 'within' logic:
  cy.get('.cx-card-title').should('contain', 'Placed on');
  cy.get('.cx-card-label').should('not.be.empty');